### PR TITLE
Adds parameter for multiple samples per trial.

### PR DESCRIPTION
**Pull request by scottpurdy**

### CODE REVIEW

**Comment by scottpurdy** [comment]
@subutai - please review

File: sdr_paper/sdr_calculations.cpp [hunk 1]
```diff
@@ -222,7 +236,7 @@ void classificationFalseMatchProbability(
     {
       if (matchesWithThetas[theta] > 0)
```
Comment (subutai) [inline sdr_paper/sdr_calculations.cpp]: We don't need the if condition do we? We can just put `probWithThetas[theta] += matchesWithThetas[theta];`

**Comment by subutai** [comment]
:+1:

File: sdr_paper/sdr_calculations.cpp (hunk 1 above)
Comment (scottpurdy) [inline sdr_paper/sdr_calculations.cpp]: good point

**Pull request merged**